src/Surfnet/StepupRa/RaBundle/Service/RaCandidateService.php

```diff
@@ -102,15 +102,16 @@ public function search(SearchRaCandidatesCommand $command)
     /**
      * @param string $identityId
      * @param string $institution
+     * @param string $actorId
      * @return null|\Surfnet\StepupMiddlewareClientBundle\Identity\Dto\RaCandidateInstitutions
      */
-    public function getRaCandidate($identityId, $institution)
+    public function getRaCandidate($identityId, $institution, $actorId)
```

Comment:
Is no additional input validation required for the actor id?

Reply:
In theory this shouldn't be needed because the actorId should be set with data from the the user, and this could be trusted because it comes from mw. I did however add a check to confirm to the security by design principle.